spec/models/missing_basket_spec.rb

```diff
@@ -9,7 +11,13 @@
     end
 
     it "returns an 'None' relation" do
-      expect(MissingBasket.new.line_items).to be none
+      expect(basket.line_items).to be none
+    end
+  end
+
+  describe "#to_partial_path" do
+    it "returns 'empty_basket'" do
+      expect(basket.to_partial_path).to eql('empty_basket')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.